migrations/versions/e79a100e0252_.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+"""empty message
+
+Revision ID: e79a100e0252
+Revises: 1cdb69cbdcaa
+Create Date: 2022-06-24 14:07:33.184539
+
+"""
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = 'e79a100e0252'
+down_revision = '1cdb69cbdcaa'
+
+
+def upgrade():
+    """
+    Upgrade Semantic Description:
+        ENTER DESCRIPTION HERE
+    """
+    # ### commands auto generated by Alembic - please adjust! ###
+    with op.batch_alter_table('asset_group_sighting', schema=None) as batch_op:
+        batch_op.alter_column(
+            'curation_start',
+            existing_type=postgresql.TIMESTAMP(),
+            nullable=True,
+        )
+
+    # ### end Alembic commands ###
+
+
+def downgrade():
+    """
+    Downgrade Semantic Description:
+        ENTER DESCRIPTION HERE
+    """
+    # ### commands auto generated by Alembic - please adjust! ###
+    with op.batch_alter_table('asset_group_sighting', schema=None) as batch_op:
+        batch_op.alter_column(
+            'curation_start',
+            existing_type=postgresql.TIMESTAMP(),
+            nullable=False,
+            existing_server_default=sa.text('CURRENT_TIMESTAMP'),
```

Comment:
Work in that I've run downgrade and upgrade a few times and it runs.
Work in that it's guaranteed that the DB is definitely right. 
Job found some other stuff that might be bad already.

Reply:
This is invoking a built-in [PostgreSQL function](https://www.postgresql.org/docs/current/functions-datetime.html#FUNCTIONS-DATETIME-CURRENT) and should work.  The downside is it drops compatibility with SQLite3 (which is no longer directly supported) and obfuscates slightly what the eventual column values will become.  An alternative way would be to use `datetime.datetime.now()`, but this will suffice.